parity/cli/mod.rs

```diff
@@ -1808,6 +1817,8 @@ mod tests {
 			arg_pruning: "auto".into(),
 			arg_pruning_history: 64u64,
 			arg_pruning_memory: 500usize,
+			arg_storage_writing: "off".into(),
+			arg_watched_contracts: Some("0xdeadbeefcafe0000000000000000000000000000".into()),
```

Comment:
should this be a default contract address? or is it just included to demonstrate the format/data type of the field?